tools/cockpit.spec

```diff
@@ -322,6 +332,7 @@ This package is not yet complete.
 %files docker -f docker.list
 %dir %{_datadir}/%{name}/docker
 
+%ifnarch armv7hl
```

Comment:
I went through a couple iterations on how to properly handle the %ifarch and %ifnarch bits. That was a leftover. I've converted it to %ifarch.